pallets/emission0/src/lib.rs

```diff
@@ -99,7 +99,10 @@ pub mod pallet {
         type WeightInfo: WeightInfo;
     }
 
+    const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
Why is this the target version and not the source version?

Reply:
we have to think that the migration only runs on chains that already exist, the `STORAGE_VERSION` exists as a value that contains the old number.

This storage version must be the target one in case you start a chain from scratch, in this case it wouldnt have a migration to updated it from 0 -> 1.

Reply:
Ok, makes sense. I can just assume this number will be `0` if not otherwise specified, correct?